base/utils/R/read.output.R

```diff
@@ -141,6 +141,11 @@ read.output <- function(runid, outdir, start.year = NA, end.year = NA, variables
       for (v in variables) {
         if (v %in% c(names(nc$var), names(nc$dim))) {
           newresult <- ncdf4::ncvar_get(nc, v)
+          if("pft.numbers" %in% names(nc$dim)){
+            # means there are PFT specific outputs we want
+            pft.ind <- strsplit(nc$var$PFT$longname, ",")[[1]] == pft.name
```

Comment:
1) Intuitively, I'd expect this code to grab all PFTs if pft.name isn't specified explicitly (default = NULL). I'm pretty skeptical that this code will do that correctly
2) nc$var$PFT is also not referencing a standard name. ```pft``` should be a dim not var
3) attribute should be long_name not longname
4) long_name should be accessed using ncdf4::ncatt_get